### ISSUE TITLE
Access Denied alert and crash

### ISSUE THREAD

**Issue by zmp2000** [SteveSandersonMS/WebWindow#103]
Hello, I'm ruinning the sample application HelloWorld
and when the program starts it shows a message "Access Denied"
![image](https://user-images.githubusercontent.com/1984798/79282989-f0e0b180-7e8c-11ea-9322-339e173bdf21.png)

After i Hit ok the follow exception is thrown:
Exception has occurred: CLR/System.AccessViolationException
An unhandled exception of type 'System.AccessViolationException' occurred in WebWindow.dll: 'Attempted to read or write protected memory. This is often an indication that other memory is corrupt.'
<Cannot evaluate the exception stack trace>

I'm running Under Windows 10 Pro Version	10.0.18363 Build 18363

**Comment by zmp2000** [comment]
Well, this have something to do with VS Code permissions, If I run the application outsite the VS Code, it runs fine, 
I think VS Code is launching it with some weird permissions, it's not creating a folder inside the application (The one the browser uses to store things)

**Comment by bretehlert** [comment]
In my experience an "Access is denied" error on Windows is caused by the failed creation of the WebView2 userDataFolder as described by https://github.com/SteveSandersonMS/WebWindow/issues/100

**Comment by musictopia2** [comment]
I tried to create the exe to run.  I even tried to do a publish and run it.   Both still gave the access denied.  I wish there was a way to fix this because I really wanted a way to run native apps with blazor and not just go to a webpage.

**Comment by ArieBen** [comment]
was this resolved?

**Comment by Smurf-IV** [comment]
I "resolved it" _temporarily_, by installing into the users `AppData`. 
Not ideal, but it allows the WebView to "do its thing".
Will wait for the WebView to start using `ProgramData` properly, as has been requested in others who have raised this issue

**Comment by musictopia2** [comment]
There is a brand new project that was based on this one here
https://www.tryphotino.io/ 
I tried this one and did not run into any problems.  Was able to run the sample project with no problems on windows desktop.  Its a good temporary solution until .net 6 comes out.

**Comment by ArieBen** [comment]
yes very interesting and it looks versatile too, looking into it... thanks a lot for the input.
